### DISCUSSION TITLE
Dashboard keeps saying "incorrect configuration" but config is OK

### DISCUSSION THREAD

**Discussion by nmeynard** [vercel/community#2379] [Help] [answered]
The dashboard has a bug, always showing "incorrect configuration" even if the config is OK (did it many times, it worked like a charm and suddenly stopped working showing this message).
When I noticed Vercel about this with screen capture demonstrating the issue they asked me to buy support credit...
Seems like some process in Vercel to make people pay to fix bugs ????

**Comment by amyegan** [comment, marked as answer]
😕  I've never heard of "support credits". I don't think that's a thing at Vercel. If you share a screenshot, I can look into it.

For the "incorrect configuration", the most common cause is that DNS change can take some time to propagate. You can check propagation status with a tools like [whatsmydns.net](https://www.whatsmydns.net/) or `dig` like in this guide: [Why is my Vercel domain not verified?](https://vercel.com/guides/why-is-my-vercel-domain-unverified)

You can find a list of other common problems and their solutions in this doc: [Troubleshooting domains](https://vercel.com/docs/concepts/projects/domains/troubleshooting#)

**Reply by nmeynard** [reply]
As I said, the configuration worked at first setup, and stopped working a few days after, so it's not a propagating issue.
When I said "support credit" I meant that Vercel ask to pay support for something that's just a bug on their side...
I also tried the two config options, none of them work (having exactly the same DNS config that the dashboard shows).
Already sent a screenshot of the dashboard and my DNS config to Vercel "support". I don't have this screenshot on this computer.

**Reply by amyegan** [reply]
I looked into the details of your support case, and it seems that you do have a problem with the DNS record. You can check the A record by running `dig passion-habitat.com +short` to check if `76.76.21.21` is returned. At the time I am writing this, the value coming back is not the correct IP address for Vercel. I recommend reaching out to your DNS provider for assistance as it seems there may be a problem on their end.

I think there was a bit of confusion about support fees. While [email support](https://vercel.com/pricing#:~:text=Community%20Support-,Email%20Support,-SLA%20for%2099.99) is a feature mostly limited to Pro and Enterprise customers, all customers do have access to support here in our community discussions. We do not charge an additional fee for support, bug reports, or questions, and I welcome you to continue to reach out here for any other help you need with Vercel. 🙂

**Comment by ErkiKadhafi** [comment]
hi i have same problem, do you already solve it?
